lib/tasks/application_forms.rake

```diff
@@ -61,4 +61,12 @@ namespace :application_forms do
       puts "#{application_form.reference}: #{application_form.status}"
     end
   end
+
+  desc "Update status of application forms in waiting on to overdue"
+  task :update_waiting_on_applications, :environment do
+    user = "Status updater"
```

Comment:
```suggestion
    user = "Expirer"
```

Since we're fixing the `ExpireRequestable` service, I think we should show that as the user which changed the status.